### PR TITLE
Art Stuff hotfix

### PR DESCRIPTION
**Pull request by Mixelz**
### Hotfix PR:
Fixes Case Sensitive bugs from PR #545.
Fixed bug where selected perk image always used Default icon set.
Defaults all images to using Image-Rendering: Pixelated, to allow new images to look pretty c: (This can be overwritten as needed)
Minor Tweaks to the Perk Menu. (No more Gold Border, slight padding changes.)
Minor Spelling Fix for "Cool QoL Cubes" name.

### CODE REVIEW

File: Synergism.css [hunk 1]
```diff
@@ -20,6 +20,10 @@ html {
     scrollbar-width: none;
 }
 
+img {
+    image-rendering: pixelated;
```
Comment (KhafraDev) [inline Synergism.css:24]: why is this needed? can you show screenshots of before/after?

File: src/EventListeners.ts [hunk 2]
```diff
@@ -23,7 +23,7 @@ import type { OneToFive, Player } from './types/Synergism'
 import { displayStats } from './Statistics'
 import { testing } from './Config'
 import { DOMCacheGetOrSet } from './Cache/DOM'
-import { toggleAnnotation, toggleTheme, toggleIconSet, imgErrorHandler } from './Themes'
+import { toggleAnnotation, toggleTheme, toggleIconSet, IconSets,  imgErrorHandler } from './Themes'
```
Comment (KhafraDev) [inline src/EventListeners.ts:26]:
```suggestion
import { toggleAnnotation, toggleTheme, toggleIconSet, IconSets, imgErrorHandler } from './Themes'
```

**Review by KhafraDev — CHANGES_REQUESTED** [review]

File: Synergism.css (hunk 1 above)
Comment (Mixelz) [reply Synergism.css:24]: This is based on the forum thread found within suggestions on the discord server, it enforces the pixelated rendering style so that the pixel art of the game does not get blurred together via anti-aliasing.
https://discord.com/channels/677271830838640680/1112301231457706064
Before and After pictures are within the thread.

Comment (KhafraDev) [reply Synergism.css:24]: thank you, it's fine - I just wanted to know what it was doing since it was being applied to all images.
